recipify/recipify/views.py

```diff
@@ -14,6 +14,10 @@
 from .util_functions.json_parser import parseRecipes
 from .util_functions.json_parser import parseIngredients
 from.util_functions.gemini_call_fun import getIngredients
+from.util_functions.image_generation import getDishImage
+from.util_functions.image_generation import add_base64_padding
+from.util_functions.image_generation import decode_base64_to_image
```

Comment:
These two are not required right? All this stuff is handled in the getDishImage function itself

Reply:
Yup, they were required by the earlier version when I was passing the image in a different way, ig let them be not an issue

Reply:
Unless they're required, why keep them. Let's not have unnecessary imports